### ISSUE TITLE
Bug:  Version 2.18.0 - Log menu item only refreshes Playlist editor

### ISSUE THREAD

**Issue by jimmymarbles** [SenexCrenshaw/StreamMaster#111]
**Describe the Bug**
Pressing the Log menu item only refreshes the Playlist Editor

**Stream Master Info**
- Version Number 2.18.0

**Link to logs (required)**
- Upload any relevant logs here [Gist](http://gist.github.com). _Do not upload attachments_.

**Comment by R0ltec** [comment]
I concur that this is happening since the last update from maybe 3-5 days ago. While not such a big deal, its nice sometimes to see what was happening prior to looking at only the Status page.

**Comment by SenexCrenshaw** [comment]
are you talking about the log menu? 

<img width="761" alt="image" src="https://github.com/SenexCrenshaw/StreamMaster/assets/35600301/2f1b5ce1-fe4f-46f0-a8ea-360d3a86eb67">

**Comment by R0ltec** [comment]
Yes correct. Somehow when we click on Log, it just goes instead to the Playlist Editor page. We are never able to see a log like that in your picture.

**Comment by SenexCrenshaw** [comment]
i see now. its only in docker. checking it out now.

**Comment by jimmymarbles** [comment]
@SenexCrenshaw You convinced me to switch over and use Docker, now there are Docker only bugs.....  Long live linux containers!

**Comment by SenexCrenshaw** [comment]
i think its more that its in a prod and not dev environment actually. not really docker
